### DISCUSSION TITLE
Gotenberg images don't seem to work on Synology DS220+

### DISCUSSION THREAD

**Discussion by Leen58** [gotenberg/gotenberg#712] [Q&A]
I have a laptop with Ubuntu 23.10 and run a docker image of gotenber latest. With a curl post i can convert any office document to pdf. 
When i do this on a synology DS220+ (celeron cpu) with exact the same image and curl post i consequently get a HTTP 503  with  a return of a non valid 19 bye PDF 
I have tried this via the Synology Docker gui and through command line docker. I suspect a problem with the gotenberg images. 
I have also tried every available previous docker gotenberg images with the same result. 
Who has the same experience and knows a solution?

**Comment by gulien** [comment]
Hello @Leen58,

My guess is that your CPU is not as powerful as your laptop. Try increasing drastically the API timeout, for instance `--api-timeout=120s`.

**Comment by Leen58** [comment]
Julien
Thanks for your reply.
That was already one of the things i had changed. The api-timeout was 600
seconds (10 minutes) but that makes no difference.
Also the test-document is half a page. On my laptop it takes milliseconds.
I think the capacity difference between the laptop and the Synology is not
that big ( 600.000 times faster)

So my guess is that there are other issues. It's also possible that the
problem lies in the Docker app on synology but thats unlikely.

Leen


Op do 2 nov 2023 om 13:38 schreef Julien Neuhart ***@***.***>:

> Hello @Leen58 <https://github.com/Leen58>,
>
> My guess is that your CPU is not as powerful as your laptop. Try
> increasing drastically the API timeout, for instance —api-timeout=120s.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/gotenberg/gotenberg/discussions/712#discussioncomment-7456320>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQYSHHXQHHVIQFSZQGDWB23YCOH5RAVCNFSM6AAAAAA62XT65CVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TINJWGMZDA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
M.vr.gr
Leen

**Reply by gulien** [reply]
I believe it’s an `amd64` platform? And the CPU is not really powerful?

I do remember that LibreOffice does struggle if it has not enough CPU power at its disposal. Given you have also Docker running, it may not have enough memory nor CPU. But that’s a supposition, I’ve no idea actually 😅
